sql/2020/22_HTTP_2/percent_of_resources_loaded_over_h2_or_h1_1_per_site.sql

```diff
@@ -0,0 +1,29 @@
+# standardSQL
+# Percentage of resources loaded over HTTP/2 or HTTP/1.1 per site
+SELECT
+  client,
+  APPROX_QUANTILES((http_1_1/num_requests) * 100, 1000)[OFFSET(500)] AS median_avg_pct_http_1_1,
```

Comment:
Ditto on percentiles.

Reply:
@rviscomi  I actually removed the percentiles from the query logic. Thoughts?